### ISSUE TITLE
[Bug] Maple tweak issue

### ISSUE THREAD

**Issue by Se7Touch** [CreatureSurvive/Fabric-Issues#18]
Please add support Maple lock screen animation with fabric battery statatus bar.

**Comment by CreatureSurvive** [comment]
What issues are you having with it?

**Comment by Se7Touch** [comment]
If enabled fabric battery status option,i plug device no playing maple charge animation and i see only blank screen after charging.

**Comment by CreatureSurvive** [comment]
So I just did some testing. I installed Maple, and it worked fine, it showed the charging animation and banners correctly. Then I enabled `Show Charging Animation` in Maple (it was disabled by default) and then it stopped show the animation. Then I disabled fabric, and it still would not show the animation. Then I uninstalled Fabric and it still would not show the animation. So finally I decided to disable the `Show Charging Animation` option in Maple again, and it still won't show, even with Fabric uninstalled. 

This seems more like a Maple issue than a Fabric issue, as the same problem you described is happening even with Fabric not installed.

I'd recommend contacting LaughingQuoll to see if he is aware of this issue.
